### ISSUE TITLE
Can't open my cp1 and the power yellow light flashes

### ISSUE THREAD

**Issue by Yu2erer** [HappyZ/dpt-tools#67]
I want to recover my cp1 system. So I download the official pkg and unpack the pkg to get the system.img. Then I use the tools to flash it. However when it ask me it's a sparse img?  I ask no and reboot. Now it can't open and the power yellow light flashes. How can I do now?

**Comment by yefeiyu** [comment]
Could you enter the `--diagnosis` mode ?

**Comment by Yu2erer** [comment]
I can't enter.  Can't see a black square in the middle.
2019年2月24日 +0800 22:30 yefeiyu <notifications@github.com>，写道：

**Comment by nyanadassana** [comment]
This happens to my rp1 once, you can do hard reset, then try to go diagnose mode again and flash again. I recommend you flash the modified pkg.  try read #62

**Comment by Yu2erer** [comment]
Press the reset button？
2019年2月24日 +0800 23:35 nyanadassana <notifications@github.com>，写道：

**Comment by nyanadassana** [comment]
There is Rest hole near the power button

**Comment by Yu2erer** [comment]
But the tools readme note Don't to press the reset button. It will make thing even worse.

2019年2月25日 +0800 00:08 nyanadassana <notifications@github.com>，写道：

**Comment by HappyZ** [comment]
Are you able to press HOME key before the third bar appear and enter the diagnosis mode?

Turn off the device. Then hold HOME key, and pressed POWER key. You released POWER, then yellow light flashes, and you keep holding HOME till the third bar on the top left corner appears.

Then the three bars should turn solid block and center block appears. 

If so, you are in the diagnosis mode.

**Comment by Yu2erer** [comment]
What is the third bar? I pressed the home key and the power key but no thing happened.
2019年2月25日 +0800 06:22 HappyZ <notifications@github.com>，写道：

**Comment by HappyZ** [comment]
> What is the third bar? I pressed the home key and the power key but no thing happened. 2019年2月25日 +0800 06:22 HappyZ <notifications@github.com>，写道：
> […](#)
> Are you able to press HOME key before the third bar appear and enter the diagnosis mode? — You are receiving this because you authored the thread. Reply to this email directly, view it on GitHub, or mute the thread.

Have you patched the update script after I update 2/9/2019? If so, you should see that while you try to boot into diagnosis mode.

If you only have yellow light flashes, but nothing appears, it means it's trying to get into diagnosis but failed.

If you are able to boot into normal system (but can't get into system), and have my modified update script before 2/9/2019, and also you have modified boot image with mass storage, then you should have mass storage appeared. 

Then put my modified official PKG into it and reboot. See if you can get through the update process.

**Comment by Yu2erer** [comment]
I only have yellow light flashes and I can't enter the diagnosis mode. No response when I pressed all key. The old script enter diagnosis mode should press the power button once, then press the home key. But now there are not responding.
2019年2月25日 +0800 06:55 HappyZ <notifications@github.com>，写道：

**Comment by Yu2erer** [comment]
I press the reset button. Now I enter the diagnosis mode. I try to flash your pkg later.
2019年2月25日 +0800 06:55 HappyZ <notifications@github.com>，写道：

**Comment by HappyZ** [comment]
Oh I see.. yours just didn't turn off previously so buttons didn't response at all.. pressing the reset helps turn it off properly first.. good luck on the rest of it

**Comment by Yu2erer** [comment]
I can't enter diagnosis mode now! When I pressed the power key and the home key to connect to my PC(Windows 10), it said Digit Paper connect.However, the DPT doesn't display a black square and I unable to use the tools to enter diagnosis mode shell. At last I found a repairman in Taobao to help me repair my device. It may take a week to know the result.

**Comment by HappyZ** [comment]
@yefeiyu did they fix it yet? would be curious about how, if so

**Comment by Yu2erer** [comment]
Updated: My device has been repaired by a store in Taobao. I can tell you the information that I am currently getting. I can't enter diagnosis mode but I found that I press the power key and wait a few seconds, the power key will flash yellow light then press the home key. My pc will display it connects to digit paper. After that A COM3 will be displayed on my pc. However when I enter diagnosis mode, my pc will show it connect to the serial port of COM5.  This key combination needs to be repeated serveral times to reproduce.
2019年3月2日 +0800 08:19 HappyZ <notifications@github.com>，写道：
